### ISSUE TITLE
Master branch fails install

### ISSUE THREAD

**Issue by juancentro** [imanuelcostigan/RSQLServer#11]
`devtools::install_github('imanuelcostigan/RSQLServer')`
```
Downloading github repo imanuelcostigan/RSQLServer@master
Installing RSQLServer
"C:/R/R-31~1.2/bin/x64/R" --vanilla CMD INSTALL  \
  "C:/Users/xxx/AppData/Local/Temp/Rtmpw1Jy5f/devtools7287d683aae/imanuelcostigan-RSQLServer-6453ff7"  \
  --library="C:/R/R-3.1.2/library" --install-tests 

Loading required package: bitops
* installing *source* package 'RSQLServer' ...
** R
Error in .install_package_code_files(".", instdir) : 
files in 'C:/Users/xxx/AppData/Local/Temp/Rtmpw1Jy5f/devtools7287d683aae/imanuelcostigan-RSQLServer-6453ff7/R' missing from 'Collate' field:
  src-sqlserver.R
ERROR: unable to collate and parse R files for package 'RSQLServer'
* removing 'C:/R/R-3.1.2/library/RSQLServer'
* restoring previous 'C:/R/R-3.1.2/library/RSQLServer'
Error: Command failed (1)
```

Maybe I should try installing the dplyr branch?

**Comment by imanuelcostigan** [comment]
I'll look into it. Please install from CRAN: `install.packages("RSQLServer")`.

**Comment by imanuelcostigan** [comment]
Fixed with e0a1d78
